build.ps1

```diff
@@ -10,34 +10,36 @@ if (Test-Path Env:CI) { $version = $Env:APPVEYOR_BUILD_VERSION }
 if (Test-Path Env:CI) { $configuration = $Env:CONFIGURATION }
 
 $components = @{
-    'NRules' = @{
-        name = 'NRules'
+    'NRules'                     = @{
```

Comment:
@gritcsenko what's up with these alignment changes? Is this also built-in VS linting?

Reply:
Fixed